### ISSUE TITLE
make browsable version of the service catalogue

### ISSUE THREAD

**Issue by monkeypants** [AusDTO/disco_layer#48]
following #45, make some views so the service catalogue can be browsed.

seed "disco service" for the spider - these pages should be indexed (and boosted!).

**Comment by nokout** [comment]
Overlaps with @maxious work.

**Comment by monkeypants** [comment]
yeah, sort-of. The service_catalogue is not browsing the search index, it's browsing the service catalogue itself.

This ticket should have been more explicit (by bad). It's about a "service oriented search facet":

 * boost pages that are identified as "entry points" in the service catalogue
 * use service facet to drill down through the service tree, to get to pages that are listed as entry point pages in the service catalogue plus "more like this" pages from the global index.
